app/build.gradle

```diff
@@ -101,7 +102,7 @@ dependencies {
 
     implementation 'com.davemorrissey.labs:subsampling-scale-image-view-androidx:3.10.0'
     implementation 'com.github.MikeOrtiz:TouchImageView:3.1.1'
-    implementation 'com.github.ARK-Builders:ark-filepicker:ae2c37ea87'
+    implementation 'com.github.ARK-Builders:ark-filepicker:fde52fe7a5'
```

Comment:
We should not forget to change this after merge in `ark-filepicker` repo.

Reply:
Yes yes